### DISCUSSION TITLE
How do I create and use my own redirect resource for `$redirect` filters?

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#2982] [Q&A] [answered]
Let's say I would like to redirect `document` connection to the site `example.com` towards my own redirect resource. The redirect resource is hypothetically hosted on `https://github.com/partingscientist/resources/main/resources/redirect.html`. The content of `redirect.html` is similar to `noophtml`, for example.

1. Do I add `https://github.com/partingscientist/resources/main/resources/` or `https://github.com/partingscientist/resources/main/resources/redirect.html` into my `userResourcesLocation`?
2. I would imagine that the filter I will be using is `||example.com^$doc,redirect=<token>`. Is this correct? If yes, what value should the token be filled with? `redirect.html`?
3. Can I create alias name for my resource? If yes, how would I do that?

Thank you.

**Comment by gorhill** [comment, marked as answer]
This is the legacy format which I think is still properly parsed: https://github.com/uBlockOrigin/uAssets/blob/15edbd0015aef1749c1eea8f111eaaffb091b59f/filters/resources.txt

**Reply by partingscientist** [reply]
I'll tinker with this. Thank you.
